### DISCUSSION TITLE
请问 VuePress 1.x 的代码复制插件，如何配置国际化？

### DISCUSSION THREAD

**Discussion by rqh656418510** [vuepress-theme-hope/vuepress-theme-hope#2011] [Q&A] [answered]
按照[官方文档](https://vuepress-theme-hope.github.io/v1/copy-code/zh/config.html#locales)配置代码复制插件后，国际化无法生效，版本信息如下：
- "vuepress": "^1.9.7"
- "@vuepress/theme-vue": "^1.9.7"
- "@mr-hope/vuepress-plugin-copy-code": "^1.30.0"

**Comment by Mister-Hope** [comment]
把你的配置贴出来

**Comment by Mister-Hope** [comment, marked as answer]
Locked as no response. Further information could be asked at v1 repo
